src/methods/traverse.method.ts

```diff
@@ -11,6 +11,36 @@ export function traverse(node: Node, forApp: boolean, depth = 0, webp = false):
   Array.from(Array(node.childNodes.length).keys())
     .map(i => node.childNodes[i])
     .forEach(child => {
+      try {
+        //  For InforWars.  The embedding code is a div followed by a script
+        // The code replaces it with a single HTTPS link and then sends it 
+        // to a(). 
+        let scriptNode;
+        let scriptSrc : string | null;
+        let dataVideoId : string | null;
+        if (child.nodeName.toLowerCase() === 'div' && (scriptNode=child.nextSibling) 
+          && scriptNode.nodeName.toLowerCase() === 'script'
+          // simply fail this check if getAttribute is not available in this version of node
+          // @ts-ignore
+          && !!scriptNode['getAttribute']
+          // @ts-ignore
+          && (scriptSrc=scriptNode.getAttribute('src'))
+          && scriptSrc === "https://infowarsmedia.com/js/player.js"
```

Comment:
Never embed 3rd party js into renderer please, it defeats the purpose of sanitization and secure rendering.

Reply:
Also separating each new embed into separate PR would be great, so they don't block each other if one of them has an issue.

Reply:
You have permission to create branch on this repo, feel free to do so, it will help us to test your changes locally by switching branch and testing in some existing apps like ecency.com or ecency mobile apps.